esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -851,28 +949,44 @@ EsdlBindingImpl::EsdlBindingImpl()
 {
 }
 
-EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg,
-                                 const char *binding,
-                                 const char *process) : CHttpSoapBinding(cfg, binding, process)
+EsdlBindingImpl::EsdlBindingImpl(IPropertyTree* cfg, const char *binding,  const char *process) : CHttpSoapBinding(cfg, binding, process)
 {
     m_bindingName.set(binding);
     m_processName.set(process);
 
     try
     {
-        DBGLOG("ESDL Binding %s is subscribing to all /ESDL/Bindings/Binding dali changes", binding);
-        //Since it seems we cannot subscribe to a specific /ESDL/Bindings/Binding, all bindings have to subscribe to the /ESDL/Bindings branch
-        m_pBindingSubscription.clear();
-        m_pBindingSubscription.setown( new CESDLBindingSubscription(this) );
+        char currentDirectory[_MAX_DIR];
```

Comment:
Current directory may be best.  Other option would be to 1. find the current component dir using the directory settings (there are routines somewhere to manipulate that).  2. use something like /var/HPCCSystems/hpcc-data/esdl/instance 

Could check how roxie decides where to save DALI based state.